src/app/shared/services/layout.service.ts

```diff
@@ -7,8 +7,11 @@ import { delay } from 'rxjs/operators';
 })
 export class LayoutService {
   // Service allowing modifications of the layout
+  private platformAsLTIProvider = location.hash.includes('asLTIProvider');
+  readonly hideLeftMenu = this.platformAsLTIProvider;
+  readonly hideTopRightControls = this.platformAsLTIProvider;
 
-  private fullFrameContent = new BehaviorSubject<boolean>(false);
+  private fullFrameContent = new BehaviorSubject<boolean>(this.hideLeftMenu);
```

Comment:
Hum it seems I forgot to submit the main remark in the previous review 😔
So it was:
My bad, the spec was not very clear but the bare state is: everything like full frame content, but with top-right hidden as well and impossibility to leave the full frame mode.
So I don't like the way there is are two independent (but not independent at all) states: `fullFrameContent` and `hideLeftMenu`. 
As I always try to avoid as much as possible non-minimal state and dependency between "instance" variables, I would say that "forbid leaving full frame" and "hide top right" should just be an option of the full frame content (so that "hide top right" does not make sense with the menu, which prevent having to handle this case properly in the design)

Reply:
While "forbid leaving full frame" as a `fullFrameContent` option makes sense, I doubt "hide top right" is meaningful.
I don't understand why hiding those top right controls has anything to do with full frame mode, as I understand it here we want to hide top right controls because the platform is run as LTI provider, and as it happens we want to force full frame for the same reason, but I don't think it makes the 2 states/variables dependants on each other